Makefile

```diff
@@ -88,15 +89,32 @@ docker-package: Makefile ## Builds the docker image to distribute
 	-f build-tools/Dockerfile build-tools
 	@echo "Successfully built image with tag $(REGISTRY)/$(REGISTRY_ORG)/spinnaker-operator:$(VERSION)"
 
+.PHONY: docker-package-ubi
+docker-package-ubi: Makefile ## Builds the ubi image to distribute
+	@echo "Packaging final docker image"
+	@docker build \
+	-t $(REGISTRY)/$(REGISTRY_ORG)/spinnaker-operator:$(VERSION)-ubi \
+	--build-arg BUILDER=docker-local/$(REGISTRY_ORG)/spinnaker-operator-builder:$(VERSION) \
+	--build-arg CACHE_DATE=$(shell date +%s) \
+	-f build-tools/Dockerfile.ubi build-tools
+	@echo "Successfully built image with tag $(REGISTRY)/$(REGISTRY_ORG)/spinnaker-operator:$(VERSION)-ubi"
+
 .PHONY: docker-push
 docker-push: ## Pushes the docker image to the docker registry with the full "version" tag
 	@docker push $(REGISTRY)/$(REGISTRY_ORG)/spinnaker-operator:$(VERSION)
+	@docker push $(REGISTRY)/$(REGISTRY_ORG)/spinnaker-operator:$(VERSION)-ubi
 
 .PHONY: docker-push-dev
 docker-push-dev: ## Pushes the docker image under "dev" tag
 	@docker tag $(REGISTRY)/$(REGISTRY_ORG)/spinnaker-operator:$(VERSION) $(REGISTRY)/$(REGISTRY_ORG)/spinnaker-operator:dev
 	@docker push $(REGISTRY)/$(REGISTRY_ORG)/spinnaker-operator:dev
 
+.PHONY: docker-push-ubi
+docker-push-ubi: ## Pushes the ubi image
+	@docker tag $(REGISTRY)/$(REGISTRY_ORG)/spinnaker-operator:$(VERSION)-ubi $(REDHAT_REGISTRY)/ospid-c671a98d-4965-4a3e-a945-296e36395c20/spinnaker-operator:$(VERSION)-ubi
+	@docker push $(REGISTRY)/$(REGISTRY_ORG)/spinnaker-operator:$(VERSION)-ubi
+	@docker push $(REDHAT_REGISTRY)/ospid-c671a98d-4965-4a3e-a945-296e36395c20/spinnaker-operator:$(VERSION)-ubi
```

Comment:
Is this UUID sensitive in some way?

Reply:
That UUID is an identifier of the certification project.
It is used in a similar way to an organization in the docker hub.
I'll keep it in a GitHub secret